ocs_ci/ocs/amq.py

```diff
@@ -0,0 +1,244 @@
+
+"""
+AMQ Class to run amq specific tests
+"""
+import logging
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.ocp import switch_to_default_rook_cluster_project
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.ocs import constants
+from subprocess import run, CalledProcessError
+from ocs_ci.utility.utils import run_cmd, TimeoutSampler
+from ocs_ci.utility import templating
+
+
+log = logging.getLogger(__name__)
+
+class AMQ(object):
+    """
+      Workload operation using AMQ
+    """
+
+    def __init__(self, **kwargs):
```

Comment:
**kwargs is good when we dont know about the function we are calling, lets use named param and document them in args.

Reply:
same as Ripsaw, should changed in both places, resolving for now.